lib/cli/ui/widgets/status.rb

```diff
@@ -0,0 +1,55 @@
+# frozen-string-literal: true
+require('cli/ui')
+
+module CLI
+  module UI
+    module Widgets
+      class Status < Widgets::Base
+        ARGPARSE_PATTERN = %r{
+          \A (?<succeeded> \d+)
+          :  (?<failed>    \d+)
+          :  (?<working>   \d+)
+          :  (?<pending>   \d+) \z
+        }x # e.g. "1:23:3:404"
+
+        OPEN  = Color::RESET.code + Color::BOLD.code + '[' + Color::RESET.code
+        CLOSE = Color::RESET.code + Color::BOLD.code + ']' + Color::RESET.code
+
+        SPINNER_STOPPED = CLI::UI::Spinner::RUNES[4]
+
+        def render
+          #   [          0✓               2✗             3⠼              4⌛︎           ]
+          "#{OPEN}#{succeeded_part} #{failed_part} #{working_part} #{pending_part}#{CLOSE}"
+        end
+
+        private
+
+        def zero?(num_str)
+          num_str == '0'
+        end
+
+        def colorize_if_nonzero(num_str, glyph, color)
```

Comment:
`glyph` vs `rune` seems to differentiate having colour embedded or not.  This is the version without colour (rune), so that it takes on the colour provided.  `glyph` vs `rune` doesn't communicate that difference well, making this an easy mistake to make. Perhaps a rename to make the difference more clear?